cruise-control/src/yaml/responses/progressResult.yaml

```diff
@@ -8,23 +8,4 @@ ProgressResult:
       type: integer
       format: int32
     progress:
-      $ref: '#/StepProgress'
-
-StepProgress:
```

Comment:
Can we put the definition of `ProgressResult`, `OperationProgress`,`StepProgress` in `ProgressResult.yaml`?
Because latter two are the type of `ProgressResult`'s field and they are only used in `ProgressResult`.